prow/plugins/trigger/pr.go

```diff
@@ -147,16 +155,22 @@ I understand the commands that are listed [here](https://go.k8s.io/bot-commands)
 %s
 </details>
 `
-	owner := pr.Base.Repo.Owner.Login
-	name := pr.Base.Repo.Name
+	org, repo, author := orgRepoAuthor(pr)
 	var more string
-	if owner != trustedOrg {
-		more = fmt.Sprintf("or [%s](https://github.com/orgs/%s/people) ", owner, owner)
+	if trigger != nil && trigger.TrustedOrg != "" {
+		more = fmt.Sprintf("or [%s](https://github.com/orgs/%s/people) ", org, org)
```

Comment:
This has the same issue as above. `more` should specify `trigger.TrustedOrg` here instead of `org` which will be mentioned earlier in the comment.

Reply:
Done